pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -65,21 +65,25 @@ <h3>On-chain Transaction Info</h3>
   <div class="envelope-accordion collapse collapse-close collapse-arrow bg-base-200 border-2 border-gray-300">
     <input type="radio" name="my-accordion-1" checked="checked" />
     {{ if .IsError }}
-    <div class="collapse-title bg-red-200 text-xl font-medium">
+    <div class="collapse-title bg-red-200">
       {{ else }}
-    <div class="collapse-title text-xl font-medium">
+    <div class="collapse-title">
       {{ end }}
-        <h3 class="font-semibold text-lg">ID: {{ .ID }}</h3>
+      <div class="flex items-center gap-x-1 pb-2 lg:text-lg font-semibold">
+        {{ if eq .Direction "in" }}
```

Comment:
The accordion component uses a `radio` input to toggle open and close. When I added an icon to the title the tooltip didn't appear. When I removed the input I was able to see the tooltip, so it seems that the `checked` state is somehow blocking the tooltip from appearing in the UI. This is why I displayed the text for now since that's the quicker fix than replacing the component at the moment.